views/search.jade

```diff
@@ -7,7 +7,7 @@ block content
 
     script.
         var songs = !{JSON.stringify(songs).replace(/<\//g, "<\\/")}
```

Comment:
curious...would jade let you convert this into a helper function so it's easier to read and less error prone?
like includeEscapedJavascript() ?